front/src/modules/ui/right-drawer/components/RightDrawer.tsx

```diff
@@ -47,6 +47,17 @@ export function RightDrawer() {
     mode: OutsideClickAlerterMode.absolute,
   });
   const theme = useTheme();
+
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
```

Comment:
We have a `useScopedHotkeys` hook already implemented for this kind of use case: https://github.com/twentyhq/twenty/blob/8b7314cd39100d06e001ef602ced5a403d535ac4/front/src/modules/ui/hotkey/hooks/useScopedHotkeys.ts#L13

An example of implementation can be found here:
https://github.com/twentyhq/twenty/blob/8b7314cd39100d06e001ef602ced5a403d535ac4/front/src/modules/ui/filter-n-sort/components/FilterDropdownButton.tsx#L86-L93

Reply:
+1, we should use useScopedHotkeys and use RightDrawer scope!
Sorry @thaisguigon, you could not invent it! I'm starting documenting things on our documentation so it's easier for new joiners

Reply:
Ok, done!